hugegraph-server/Dockerfile

```diff
@@ -48,8 +48,10 @@ RUN set -x \
        curl \
        lsof \
        vim \
+       cron \
     && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
+    && service cron start
 
 # 2. Init HugeGraph Sever
```

Comment:
<img width="805" alt="image" src="https://github.com/apache/incubator-hugegraph/assets/17706099/6d8da1f5-7d1a-48f4-b6c5-72a442c3ad07">

maybe step2 could also be merged into st1?